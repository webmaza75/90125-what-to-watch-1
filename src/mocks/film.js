const film = {
  genre: [`Drama`, `Thriller`],
  title: `Aviator`,
  desc: ``,
  picture: `aviator.jpg`,
  year: 2016,
  src: ``
};

export default film;
