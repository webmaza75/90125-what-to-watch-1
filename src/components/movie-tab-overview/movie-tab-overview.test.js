import React from 'react';
import renderer from 'react-test-renderer';

import MovieTabOverview from './movie-tab-overview.jsx';
import film from '../../mocks/film.js';

it(`MovieTabOverview correctly renders`, () => {
  const {
    director,
    starring,
    rating,
    scoresCount,
    description,
    ratingLevel
  } = film;

  const tree = renderer
    .create(<MovieTabOverview
      director={director}
      starring={starring}
      rating={rating}
      scoresCount={scoresCount}
      description={description}
      ratingLevel={ratingLevel}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
