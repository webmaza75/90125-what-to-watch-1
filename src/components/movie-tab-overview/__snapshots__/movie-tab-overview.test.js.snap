// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieTabOverview correctly renders 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      6,5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        386834
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    As Earth is invaded by alien tripod fighting machines, one family fights for survival.
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Steven Spielberg
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Tom Cruise, Dakota Fanning, Tim Robbins
      </strong>
    </p>
  </div>,
]
`;
